asgi_ipc.py

```diff
@@ -1,11 +1,11 @@
 from __future__ import unicode_literals
 
-import mmap
 import msgpack
+from pathlib import Path
```

Comment:
Ah, I wasn't sure if the projects maintain compatibility (and I was, admittedly, too lazy to look it up).